hbw/app/javascript/packs/hbw/components/helpers/validations.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default WrappedComponent => (props) => {
+  const isFilled = value => value !== null && value !== undefined && `${value}`.length > 0;
```

Comment:
Add parentheses around this function body, please. It will help to read this complicated line.